source/Preprocessor.cpp

```diff
@@ -82,9 +82,12 @@ Token Preprocessor::next(LexerMode mode) {
             case SyntaxKind::NoUnconnectedDriveDirective:
             case SyntaxKind::CellDefineDirective:
             case SyntaxKind::EndCellDefineDirective:
-            default:
                 trivia.append(createSimpleDirective(token));
                 break;
+            default:
```

Comment:
There is a DEFAULT_UNREACHABLE macro to assert when you hit the default case, if this really shouldn't ever be possible.